src/main/java/io/meeds/qa/ui/pages/page/factory/space/SpaceHomePage.java

```diff
@@ -1215,4 +1214,40 @@ public void viewAllRepliesInCommentsDrawer(String comment) {
     getCommentsDrawerViewAllReplies(comment).clickOnElement();
   }
 
+  private BaseElementFacade getReactionActivityList(String activity) {
```

Comment:
Please rename this to indicate that this is a link or button 
```suggestion
  private BaseElementFacade getActivityReactionsLink(String activity) {
```

Reply:
done